### PR TITLE
add a plan

### PR DESCRIPTION
**Pull request by RuiZhou96**

### CODE REVIEW

**Review by leilakrichel — APPROVED** [review]

**Review by jeff-a-nguyen — APPROVED** [review]

**Review by FarhanMajumder — APPROVED** [review]
I like the graphs

File: doc/analysis-plan.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+    ## 10           offShelf and on Shelf   163
+    ## 11                      open ocean  4011
+    ## 12                         Pelagic 16084
+    ## 13          Seasonal Pack Ice Zone  2229
+    ## 14                           Shelf  2022
+    ## 15           Shelfbreak/open ocean   439
+    ## 16               transition region   447
+
+# possible analysis
+
```
Comment (lwjohnst86) [inline doc/analysis-plan.md]: @RuiZhou96 Don't forgot this section!

File: doc/analysis-plan.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+    g <- ggplot(marine2, color = predator_taxon)
+    g+
+      geom_boxplot(aes(x = feeding_type, y = predator_mass))+
+      ylim(0,5000)
+
+    ## Warning: Removed 11279 rows containing non-finite values (stat_boxplot).
+
+# Possible results figures
+
```
Comment (lwjohnst86) [inline doc/analysis-plan.md]: Also, while the figures are great, please add some text briefly describing some *other* figures you could include.

File: doc/analysis-plan.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+    ## 4             predacious 12394
+    ## 5 predacious/piscivorous   161
+
+    g+
+      geom_point(aes(x = temp, y = predator_mass))
+
+![](123_files/figure-markdown_strict/unnamed-chunk-6-1.png)
+
+# Possible results tables
+
```
Comment (lwjohnst86) [inline doc/analysis-plan.md]: See comment above about adding text.

**Review by FarhanMajumder — APPROVED** [review]

**Review by leilakrichel — APPROVED** [review]

**Review by jeff-a-nguyen — APPROVED** [review]

**Comment by lwjohnst86** [comment]
Great that you guys have started some analyses already! The analysis-plan.md file should have a more text based, simple description of what you'll do. If you do want to include the analyses/figures, create an `analysis-plan.Rmd` file with the code, and generate an md output from it. We can help with this next class.

**Comment by QuLogic** [comment]
The `analysis-plan.md` _was_ created from an `Rmd`; we probably should have suggested that that file also be added when we discussed it last class.

**Review by leilakrichel — APPROVED** [review]

**Review by jeff-a-nguyen — APPROVED** [review]

**Review by leilakrichel — APPROVED** [review]

**Review by FarhanMajumder — APPROVED** [review]
Nice

**Comment by lwjohnst86** [comment]
@RuiZhou96 you can merge this in if you want

**Comment by joelostblom** [comment]
@RuiZhou96 @FarhanMajumder @leilakrichel @jeff-a-nguyen Make sure to merge or close this!

**Pull request merged**